src/archanaNJun20/IndexOfExample1.java

```diff
@@ -0,0 +1,39 @@
+package archanaNJun20;
+
+public class IndexOfExample1 {
+
+	void countStringChar(String input)
+	{
+		int digitCount=0;
+		int uCount=0;
+		int lCount=0;
+		int loCount=0;
+		int spCount=0;
+
+		for(int index=0;index<input.length();index++){
+			char ch=input.charAt(index);
+			if(Character.isDigit(ch)) 
+				digitCount++;
+			else  if(Character.isLetter(ch)) {
+				lCount++;
+				if(Character.isUpperCase(ch)) 
```

Comment:
Without lCount use of this variable and else  if(Character.isLetter(ch))  block you are able to achieve the output.

Reply:
Here so many comparisons happening, which is not required. 
Like if char is digit so it will go in your all if block. which is wrong right !!
So here you need to write if, else if....else block.